nvflare/private/fed/client/client_run_manager.py

```diff
@@ -113,11 +120,10 @@ def new_context(self) -> FLContext:
         return self.fl_ctx_mgr.new_context()
 
     def send_task_result(self, result: Shareable, fl_ctx: FLContext) -> bool:
-        try:
-            self.client.push_results(result, fl_ctx)  # push task execution results
+        push_result = self.client.push_results(result, fl_ctx)  # push task execution results
+        if push_result[0] == CellReturnCode.OK:
```

Comment:
This assume only one service.
For now it is fine, if anytime we do add another service need to change this.
I personally think we should just change all codes to assume we have 1 service.

Reply:
We can make the changes for that in a separate effort. That will require changes to a lot of places. The current implementation is based on the clients can talk to multiple servers. That seems an over-engineering, since we don't really have that requirement.